lib/textris/message.rb

```diff
@@ -81,15 +81,20 @@ def parse_from(from)
     end
 
     def parse_from_dual(from)
-      if (matches = from.to_s.match(/(.*)\<(.*)\>\s*$/).to_a).size == 3 &&
-          Phony.plausible?(matches[2])
+      matches = from.to_s.match(/(.*)\<(.*)\>\s*$/).to_a
+      return if matches.size != 3
```

Comment:
magic number, what 3 represents here?

Reply:
@Lackoftactics https://github.com/visualitypl/textris/blame/ab770cb966218d6a4c2225a7a7ab360d590698ae/lib/textris/message.rb#L88

But:

```
irb(main):001:0> "Jan Matusz <+48123456789>".match(/(.*)\<(.*)\>\s*$/).to_a
=> ["Jan Matusz <+48123456789>", "Jan Matusz ", "+48123456789"]
```

So basicly it "splits" the sender string (which allows us to normalize/set the sender ID and sender name explicitly).